Tests in `vitest/test/fails` fail.

### Describe the bug

Attempted to contribute to Vitest but failed the test.
Running tests in vitest/test/fails fails.

1. ` cd vitest/test-fails `
2. ` pnpm run test `

``` bash
 FAIL  test/runner.test.ts > should fails > empty.test.ts
AssertionError: Snapshot `should fails > empty.test.ts > empty.test.ts 1` mismatched
 ❯ test/runner.test.ts:35:18
     33|         .find(i => i.includes('Error: '))
     34|         ?.trim()
     35|       expect(msg).toMatchSnapshot(file)
       |                  ^
     36|     }, 10000)
     37|   }

  - Expected   "\"Error: No test suite found in file /Users/antfu/i/vitest/test/fails/fixtures/empty.test.ts\""
  + Received   "\"Error: No test suite found in file /Users/Project/vitest/test/fails/fixtures/empty.test.ts\""

```

### Reproduction

No response

### System Info

```shell
No response
```


### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/test/fails/test/runner.test.ts
+++ b/test/fails/test/runner.test.ts
@@ -32,6 +32,7 @@ describe('should fails', async () => {
         .reverse()
         .find(i => i.includes('Error: '))
         ?.trim()
+        .replace(root, '<rootDir>')
       expect(msg).toMatchSnapshot(file)
     }, 10000)
   }

